image/templates/helm/shared/templates/02-scanner-v4-05-indexer-network-policy.yaml

```diff
@@ -63,4 +63,26 @@ spec:
   policyTypes:
   - Ingress
 {{- end }}
+{{- if ._rox.monitoring.openshift.enabled }}
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: scanner-v4-indexer-monitoring-tls
```

Comment:
Maybe a stupid question, but what happens if Central + Sensor are in the same namespace, and both enable scanner v4 / scanner v4 delegated scanning? Will there be two scanner v4 deployments in the same namespace or is this forbidden somehow?

Reply:
We ensure only one Indexer is deployed. Matcher is only relevant in Central.